builtin/mainmenu/tab_multiplayer.lua

```diff
@@ -92,17 +116,27 @@ end
 
 --------------------------------------------------------------------------------
 local function main_button_handler(tabview, fields, name, tabdata)
+	print(dump(fields))
+	local serverlist = menudata.search_result or menudata.favorites
+	
+	local function check_filters(filters, server)
```

Comment:
This function can be moved outside `main_button_handler`, as it doesn't use any upvalues. It avoids regular redefinition.